code/Controllers/OldPageRedirector.php

```diff
@@ -29,7 +29,7 @@ public function onBeforeHTTPError404($request)
         $getvars = $request->getVars();
         unset($getvars['url']);
 
-        $page = self::find_old_page($params);
+        $page = static::find_old_page($params);
```

Comment:
Can you explain how this will help? I can't see why this would make any difference

Reply:
Nevermind, I'm being dumb